sleap/gui/dataviews.py

```diff
@@ -301,6 +301,7 @@ def __init__(
         is_sortable: bool = False,
         is_activatable: bool = False,
         ellipsis_left: bool = False,
+        multiple_selection=False,
```

Comment:
Add type hint for consistency:

```suggestion
        multiple_selection: bool = False,
```